sdk/core/azure-core/samples/test_example_async.py

```diff
@@ -53,29 +53,29 @@ async def req():
 async def test_example_asyncio():
 
     request = HttpRequest("GET", "https://bing.com")
-    policies = [UserAgentPolicy("myuseragent"), AsyncRedirectPolicy()]
+    policies : Iterable[Union[AsyncHTTPPolicy, SansIOHTTPPolicy]] = [UserAgentPolicy("myuseragent"), AsyncRedirectPolicy()]
     # [START asyncio]
     from azure.core.pipeline.transport import AsyncioRequestsTransport
 
     async with AsyncPipeline(AsyncioRequestsTransport(), policies=policies) as pipeline:
         response = await pipeline.run(request)
     # [END asyncio]
-    assert pipeline._transport.session is None
+    assert pipeline._transport.session is None  # type: ignore
```

Comment:
nit: Wondering if it's worth having code in the sample that we wouldn't expect users to write (assuming this since transport is "_private". Feels more like this code belongs in a test).

Reply:
Yeah, this is more a test than something customers should care, let me remove it